src/lib.rs

```diff
@@ -92,37 +96,43 @@ impl Toasts {
 
     /// Dismisses all toasts
     pub fn dismiss_all_toasts(&mut self) {
-        for toast in self.toasts.iter_mut() {
+        for toast in &mut self.toasts {
             toast.dismiss();
         }
     }
 
     /// Shortcut for adding a toast with info `success`.
+    #[must_use]
```

Comment:
These all should not be `#[must_use]` as user may not necessarily modify them after.